dbt_sugar/core/task/base.py

```diff
@@ -173,6 +175,22 @@ def update_column_descriptions(
                         path_file, dict_column_description_to_update
                     )
 
+    def update_test_in_dbt_tests(self, model_name: str, column: Dict[str, Any]) -> None:
+        """Update a column tests in the global tests dictionary.
+
+        Args:
+            model_name (str): with the model name.
+            column (Dict[str, Any]): column information.
+        """
```

Comment:
I don't really follow what we use this function for. Do we have a central tests dictionary?

Reply:
yes, now we have an object that loads which model with which column have which tests, so we can do the statistics.